contribs/gmf/options/gmfx.js

```diff
@@ -476,6 +477,14 @@ gmfx.AuthenticationFunctionalities.prototype.default_basemap;
  */
 gmfx.AuthenticationFunctionalities.prototype.default_theme;
 
+
+/**
+ * A list of layer names that can be filtered.
+ * @type {Array.<string>|undefined}
```

Comment:
I'm looking at all the other definitions and none have the `!`.  I think the compiler requires a `?` when a value can be null.  I think this definition is okay.  What do you think?

Otherwise, if all other properties can't be null, then they would all need `!`.